ui/src/components/dynamics-form/FormItem.vue

```diff
@@ -128,7 +128,7 @@ const rules = computed(() => {
     ? props_info.value.rules.map(to_rule)
     : {
         message: errMsg.value,
-        trigger: ['blur', 'change'],
+        trigger: props.formfield.input_type === 'Slider' ? 'blur' : ['blur', 'change'],
         required: props.formfield.required === false ? false : true
       }
 })
```

Comment:
The code looks mostly correct. However, there are a few minor improvements you might consider:

1. **Destructuring**: You can destructure `props` inside the functional component to make it cleaner.
   ```javascript
   const { formfield } = props;
   ```

2. **Consistent Spacing and Indentation**: Ensure consistent spacing around operators and braces for better readability.

3. **Type Check**: If `formfield.input_type` is expected to be `'Slider'`, ensure that it's actually checked with `.includes()` since strings use a comma-separated list when split by `,`.

4. **Comments**: Consider adding comments if they don't already explain the purpose of each step clearly.

Here's an improved version:
```javascript
import { computed, ref, reactive, watchEffect } from 'vue';

export default defineComponent({
  name: 'DynamicForm',
  setup(props) {
    // Destructure props to improve readability
    const { formfield } = props;

    // ... rest of your existing code ...
  }
});
```

This doesn't contain any irregularities or significant optimizations needed based on the initial review, but these changes will enhance the maintainability of your codebase.